### PR TITLE
Add Linux VMware Workspace One UEM util exec

### PR DESCRIPTION
**Pull request by Micah-Kolide**
Quick exec addition for workspace one. This is untested, but the paths and command args are verified.

### CODE REVIEW

**Review by RebeccaMahany — APPROVED** [review]

File: pkg/osquery/table/platform_tables_linux.go [hunk 1]
```diff
@@ -49,6 +49,7 @@ func platformSpecificTables(logger log.Logger, currentOsquerydBinaryPath string)
 			allowedcmd.Lsblk, []string{"-fJp"},
 		),
 		dataflattentable.TablePluginExec(logger, "kolide_nix_upgradeable", dataflattentable.XmlType, allowedcmd.NixEnv, []string{"--query", "--installed", "-c", "--xml"}),
+		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_enroll_status", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--enroll"}),
```
Comment (directionless) [inline pkg/osquery/table/platform_tables_linux.go:52]: I wonder if it's worth vetting this command with the customer first. I'm especially curious if we should do it with or without `--enroll`

Comment (directionless) [reply pkg/osquery/table/platform_tables_linux.go:52]: (I pinged the customer)

Comment (Micah-Kolide) [reply pkg/osquery/table/platform_tables_linux.go:52]: I can pull out the `--enroll`, but as I haven't seen the output, I'm not sure if the status itself returns data in JSON. The `--enroll`, `--dependency`, and `--profile` do confirm they are JSON formatted.

Comment (Micah-Kolide) [reply pkg/osquery/table/platform_tables_linux.go:52]: There's also `--app`, which isn't described in the documentation, but is an option shown, so I'm not sure how that data would return either.

Comment (directionless) [reply pkg/osquery/table/platform_tables_linux.go:52]: I got some back from customer. Might be interesting to do each of:
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --enroll
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --dependency
* /opt/vmware/ws1-hub/bin/ws1HubUtil status --profile

Comment (directionless) [inline pkg/osquery/table/platform_tables_linux.go:52]:
```suggestion
		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_enroll", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--enroll"}),
		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_dependency", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--dependency"}),
		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_ profile", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--profile"}),

```

File: pkg/osquery/table/platform_tables_linux.go [hunk 2]
```diff
@@ -49,6 +49,10 @@ func platformSpecificTables(logger log.Logger, currentOsquerydBinaryPath string)
 			allowedcmd.Lsblk, []string{"-fJp"},
 		),
 		dataflattentable.TablePluginExec(logger, "kolide_nix_upgradeable", dataflattentable.XmlType, allowedcmd.NixEnv, []string{"--query", "--installed", "-c", "--xml"}),
+		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_enroll", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--enroll"}),
+		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_dependency", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--dependency"}),
+		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_ profile", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--profile"}),
+
```
Comment (directionless) [inline pkg/osquery/table/platform_tables_linux.go:55]:
```suggestion
		dataflattentable.TablePluginExec(logger, "kolide_wsone_uem_status_profile", dataflattentable.JsonType, allowedcmd.Ws1HubUtil, []string{"status", "--profile"}),
```

**Review by directionless — APPROVED** [review]

**Review by RebeccaMahany — APPROVED** [review]

**Pull request merged**